dist/api_data.py

```diff
@@ -1391,6 +1391,12 @@ def __ge__(self, other):
     Config('import', '', r'''
         configure import of an existing object into the currently running database''',
         type='category', subconfig=[
+        Config('compare_timestamp', 'oldest', r'''
```

Comment:
I feel `compare_timestamp` name doesn't feel good, how about just `timestamp`?

Reply:
I feel `timestamp` is too vague.